integrations/spark/openhouse-spark-runtime/build.gradle

```diff
@@ -37,6 +38,11 @@ dependencies {
   runtimeOnly "org.antlr:antlr4-runtime:4.7.1"
   antlr "org.antlr:antlr4:4.7.1"
 
+  fatJarRuntimeDependencies("com.azure:azure-storage-file-datalake:12.19.1") {
+    exclude group: 'io.netty'
```

Comment:
Please add some details in the PR description regarding the netty version brought in by these changes and the libs that are excluded. Good to add some details in PR on the iceberg azure libs are are inegrated.

Reply:
Why did we need to add this dependency?  it was not required for s3

Reply:
we've to be careful with this dependency as it will start showing up on all users classpath and may affect production flows

Reply:
I think we don't actually need it. It was left over as part of our experiments debugging the client cluter